lib/components/app/app-menu.tsx

```diff
@@ -319,14 +321,10 @@ const IconAndLabel = ({
           )}
           src={iconUrl}
         />
+      ) : iconType ? (
+        <SvgIcon iconName={iconType} />
```

Comment:
This seems to have been lost already in the font awesome pr, but there used to be support for an icon being a URL that pointed to an image. Do you think we could restore that functionality? (By checking if the `iconType` is a URL and if so rendering an `img`

Reply:
Sorry it is the `iconUrl` property we are ignoring!

Reply:
From the changes in #651, I am not sure that a check for whether an image loaded successfully was there before.

Reply:
Yeah I think my memory is playing tricks on me. I believe we just rendered the image if `iconUrl` was present, otherwise the svg tag

Reply:
If your question was regarding the fallback if `iconType` is unknown, then yes, `SvgIcon` falls back to the external link icon.

Reply:
No we used to support `iconUrl`, and `iconType` was the fallback